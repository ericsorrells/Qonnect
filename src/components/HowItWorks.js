// ========================================================================================
import React from 'react';
// ========================================================================================
import HowItWorksItem from './HowItWorksItem';
// ========================================================================================

const HowItWorks = () => {
  return (
    <div>
      How It Works Component!!
      <HowItWorksItem />
    </div>
  )
}

export default HowItWorks;