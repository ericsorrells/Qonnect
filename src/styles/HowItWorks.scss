.how-it-works__container {
  background-image: linear-gradient(to top, $main-color-light, $main-color);
  background-color: #534292;
  padding-top: 50px;
  padding-bottom: 50px;
}

.how-it-works__inner-container {
  display: flex;
  justify-content: space-around;
}