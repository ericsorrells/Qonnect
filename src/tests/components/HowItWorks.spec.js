// ========================================================================================
import React from 'react';
import { shallow } from 'enzyme';
import 'jest-enzyme';
// ========================================================================================
import HowItWorks from '../../components/HowItWorks';
// ========================================================================================

describe('<HowItWorks />', () => {
  const component = shallow(<HowItWorks/>)

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });

  it('renders all three child elements', () => {
    expect(component.find('.how-it-works__inner-container').children().length).toBe(3);
  });
});
