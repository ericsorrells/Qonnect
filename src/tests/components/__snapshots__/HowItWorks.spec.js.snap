// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HowItWorks /> renders correctly 1`] = `
<div
  className="how-it-works__container"
>
  <div
    className="how-it-works__inner-container"
  >
    <HowItWorksItem
      icon="fas fa-calendar-alt"
      text="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
      title="Find Something To Do"
    />
    <HowItWorksItem
      icon="fa fa-users"
      text="Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      title="Meet New People"
    />
    <HowItWorksItem
      icon="fas fa-hands-helping"
      text="Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit"
      title="Find Something To Do"
    />
  </div>
</div>
`;
